### ISSUE TITLE
Unable to use "@github subscribe" command when another GitHub App is installed in MS Teams

### ISSUE THREAD

**Issue by ppremk** [integrations/microsoft-teams#47]
### TLDR;

Having another installed GitHub App by a 3rd party developer with the same name, does not allow for subscribing repo notifications 

### Details
This could be a potential bug. I had looked at the closed and open issues and was not able to see a previous note on this behaviour of the app. Apologies if this is a known issue.

**Context (Steps to reproduce):**
1. Installed the wrong `GitHub` App from the App tab from a private channel

<img width="908" alt="Screenshot 2020-09-18 at 00 32 06" src="https://user-images.githubusercontent.com/5770369/93535328-0891f980-f947-11ea-8e76-d20b9f0e1b15.png">

2. Remove the wrong GitHub App from the private channel

3. Install the correct `GitHub (Preview)` App to the same private channel

<img width="413" alt="Screenshot 2020-09-18 at 00 31 33" src="https://user-images.githubusercontent.com/5770369/93535327-07f96300-f947-11ea-9b4a-b3279237768a.png">

4. Follow standard installation steps. Authorize App to orgs and accounts. Verify App is install in Orgs and accounts on GitHub

5. Type in `@github subscribe` to start monitoring notifications. While typing, notice that both `GitHub` apps are showing up in the preview

<img width="933" alt="Screenshot 2020-09-18 at 00 31 08" src="https://user-images.githubusercontent.com/5770369/93535326-07f96300-f947-11ea-92b8-d5529e964d5e.png">

6. If we explicitly type `@github` we do not get the subscribe option. This somehow defaults and falls back to the wrongly installed app. This was misleading since there was no reply or feedback when the wrong command was typed in

<img width="942" alt="Screenshot 2020-09-18 at 00 32 38" src="https://user-images.githubusercontent.com/5770369/93535329-0891f980-f947-11ea-9852-ff9e6665662d.png">

**Workaround:**

1. While typing `@github`, when the pop up appears with the `GitHub (Preview)` app option, select the correct app from the pop up then continue to type the subscribe or unsubscribe to use the correct version of the app

**Comment by ashokirla** [comment]
@ppremk Firstly, can you check if you have uninstalled the old GitHub app from the channel, personal app (on sidebar) and compose extension (bottom of the message box ...). This could be the reason why you are seeing old github bot when you type @ github.
In Teams, once you type @ github, you need to select the GitHub (Preview) bot from contextual popup before running any command. This is the Teams behavior for informing bot to perform actions. Directly typing the command @ github <command> will not work.
We have shared this feedback to Teams platform team.

**Comment by ppremk** [comment]
@ashokirla 

Thank you for the fast response 🙇 

> can you check if you have uninstalled the old GitHub app from the channel, personal app (on sidebar) and compose extension (bottom of the message box ...). This could be the reason why you are seeing old github bot when you type @ github.

It is uninstalled now. However for scenarios for having both the Apps installed, how are we going to differentiate this since it shows up as the same name? Now the differentiator is within the (Preview) brackets 🙂 

> In Teams, once you type @ github, you need to select the GitHub (Preview) bot from contextual popup before running any command. This is the Teams behavior for informing bot to perform actions. Directly typing the command `@github command` will not work.

TIL. Perhaps having this as a note in the [documentation](https://github.blog/2020-09-10-announcing-the-github-integration-with-microsoft-teams/#customizing-your-experience) will help folks whom are switching from slack to MS Teams 🙂 


> We have shared this feedback to Teams platform team.

Awesome Thank you ❤️

**Comment by ashokirla** [comment]
The old app (TCS) name is now change to `GitHub (Legacy)` to avoid confusion.

We will update documentation to make it clear about running comments in Teams.
